lib/monkey/include/monkey/mk_core/mk_bucket_queue.h

```diff
@@ -0,0 +1,129 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
```

Comment:
Not sure what license to use? Just copied from some other monkey lib.
Maybe we can add AWS somewhere in there.